sdk/python/tests/test_online_retrieval.py

```diff
@@ -110,6 +112,29 @@ def test_online() -> None:
         assert result["customer_profile__name"] == ["John", "John"]
         assert result["customer_driver_combined__trips"] == [7, 7]
 
+        # Test the dataframe conversion
+        result_df = store.get_online_features(
+            feature_refs=[
+                "driver_locations:lon",
+                "customer_profile:avg_orders_day",
+                "customer_profile:name",
+                "customer_driver_combined:trips",
+            ],
+            entity_rows=[{"driver": 1, "customer": 5}, {"driver": 1, "customer": 5}],
+        ).to_df()
+        expected_df = pandas.DataFrame(
+            {
+                "driver": [1, 1],
+                "customer": [5, 5],
+                "driver_locations__lon": ["1.0", "1.0"],
+                "customer_profile__avg_orders_day": [1.0, 1.0],
+                "customer_profile__name": ["John", "John"],
+                "customer_driver_combined__trips": [7, 7],
+            }
+        )
+        # Ignore the order of columns
+        assert_frame_equal(result_df, expected_df, check_like=True)
```

Comment:
Can we try to use a few different entities? I want to make sure that we are testing for ordering as well (ideally the response rows would be in the same order as the request).

Reply:
It seems OnlineResponse obj does not maintain the orders of the original request.  Perhaps I can pass the feature_refs and entity_rows into the the OnlineResponse and then sort the col and rows afterwards?

Reply:
> It seems OnlineResponse obj does not maintain the orders of the original request. Perhaps I can pass the feature_refs and entity_rows into the the OnlineResponse and then sort the col and rows afterwards?

Ideally we would push this responsibility down to the connectors and wouldn't sort all responses for them, otherwise we will always pay a performance penalty. What do you think about adding a test for consistency of ordering here and then we can create a new issue/PR to make sure ordering is upheld in responses from the storage connectors?

Reply:
Done. Please take a look if I understood correctly, thanks.